src/main/java/org/openpnp/vision/pipeline/CvPipeline.java

```diff
@@ -197,6 +198,14 @@ public void setNozzle(Nozzle nozzle) {
     public Nozzle getNozzle() {
         return nozzle;
     }
+	
+    public void setValue(double value) {
```

Comment:
ok, i see, i need to remove closed loop and other functionality for having more simplier code